### ISSUE TITLE
Game Over Screen

### ISSUE THREAD

**Issue by patricky95** [wonderlili/wonderlili.github.io#38]
When you lose, it shows you the high score and your score. The score x coordinates are large enough, and consequentially the score is inside the text.

**Comment by patricky95** [comment]
I would also like to mention that there should be a way to either move along the x axis, or shrink the diameter randomization of the blobs because sometimes it is impossible to pass them. (Unless of course that is intended)

**Comment by wonderlili** [comment]
I will shrink the max diameter. It is slightly intended, but I think that "a change of decorations is in order". I'll see how that works. Any recommendations for a new max diameter?

**Comment by wonderlili** [comment]
Also can you elaborate on your first comment? Thanks.

**Comment by patricky95** [comment]
Yes. Basically when it's game over (or if you win I haven't checked) it will show you your score and high score. The actual score number overlaps the text that says "score" and "high score".
